build-hpx.sh

```diff
@@ -45,7 +45,8 @@ if [[ ! -d ${DIR_SRC} ]]; then
 	cd ..
         git clone https://github.com/STEllAR-GROUP/hpx.git
 	cd hpx
-	git checkout 1.6.0
+	git checkout 1.7.1
```

Comment:
Instead of hard coding 1.7.1 we should use 

export HPX_VERSION=1.5.1

in config.sh and bump the version there.